docs/integrations/emr/emr_configuration.md

```diff
@@ -0,0 +1,215 @@
+# Configuring EMR for the Hopsworks Feature Store
+To enable EMR to access the Hopsworks Feature Store, you need to set up a Hopsworks API key, add a bootstrap action and configurations to your EMR cluster.
+
+!!! info
+    Ensure [Networking](networking.md) was set up correctly before proceeding with this guide.
+
+## Step 1: Setting up a Hopsworks API Key
+In order for EMR clusters to be able to communicate with the Hopsworks Feature Store, the clients running on EMR need to be able to access a Hopsworks API Key.
+
+### Generating an API Key
+
+In Hopsworks, click on your *username* in the top-right corner and select *Settings* to open the user settings. Select *Api keys*. Give the key a name and select the project scope before creating the key. Copy the key into your clipboard for the next step.
+
+!!! success "Scopes"
+    The created API-Key should at least have the following scopes:
+
+    1. project
+
+<p align="center">
+  <figure>
+    <img src="../../../assets/images/emr/api_key.png" alt="Generating an API Key on Hopsworks">
+    <figcaption>API-Keys can be generated in the User Settings on Hopsworks</figcaption>
+  </figure>
+</p>
+
+!!! info
+    You are only able to retrieve the API Key once. If you miss to copy it to your clipboard, delete it again and create a new one.
+
+### Storing the API Key in the AWS Secrets Manager
+
```

Comment:
Not the case for EMR, the secret is needed to boot the cluster, the only alternative is to put it in the bootstrap arguments and it will show up all over in the logs. I had this option first but it complicates things a lot. The different options make it confusing what to do, you'll have to supply different arguments or scripts and the keys will certainly be exposed without the user noticing.